README.md

```diff
@@ -235,13 +258,17 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
+| <a href="https://www.drawkit.com/product/startups-and-saas-icons"><img width='700' src='https://user-images.githubusercontent.com/34191321/194964526-e758799c-320c-422e-8a16-f27b65b2ce19.png' alt="startups_and_saas_icons_free_illustrations" /></a> | [Startups and SaaS Icons](https://www.drawkit.com/product/startups-and-saas-icons)  <br><br> ✔ Free <br><br>Format <br> • SVG<br> • PNG<br>• JPG<br><br> ✌️Author <br> Drawkit                |
 | <a href="https://www.streamlinehq.com/illustrations/free-illustrations-bundle"><img width='700' src='https://user-images.githubusercontent.com/74038190/184809664-a30ad560-c9e5-4e81-b3df-30113a624952.png' alt="Streamlinehq_free_illustrations" /></a> | [Streamline HQ](https://www.streamlinehq.com/illustrations/free-illustrations-bundle)  <br><br> ✔ Free <br><br>Format <br>• PNG<br>• SVG<br>• PDF<br><br> ✌️Author <br> Streamlinehq            |
 | <a href="https://www.figma.com/community/file/883778082594341562"><img width='700' src='https://user-images.githubusercontent.com/74038190/184642936-50c41348-3663-436e-bab6-5233d306b960.png' alt="Surface_free_illustrations" /></a> | [Surface 75](https://www.figma.com/community/file/883778082594341562)  <br><br> ✔ Free <br><br>Format <br> • Figma<br><br> ✌️Author <br> Treetop                                                             |
 | <a href="https://drawkit.com/product/team-work-illustrations"><img width='700' src='https://user-images.githubusercontent.com/92253071/193478432-a4e0fd47-1633-4b39-a680-d5f936b96753.png' alt="Team_work_free_illustrations" /></a> | [Team Work](https://drawkit.com/product/team-work-illustrations)  <br><br> ✔ Free <br><br>Format <br> • Figma<br> • PNG<br>• SVG<br><br> ✌️Author <br> Drawkit                                                     |
 | <a href="https://craftwork.design/downloads/tiny-dribbble-shots-constructor"><img width='700' src='https://user-images.githubusercontent.com/34191321/194421585-c93825ae-8d3f-4caf-88ec-0db5e359380d.png' alt="Tiny_Dribbble_Shots_Constructor_free_illustrations" /></a> | [Tiny Dribbble Shots Constructor](https://craftwork.design/downloads/tiny-dribbble-shots-constructor)  <br><br> ✔ Free <br><br>Format <br>• Figma<br><br> ✌️Author <br> Craftwork | 
+| <a href="https://drawer.design/products/ugly-delicious/"><img width='700' src='https://user-images.githubusercontent.com/74038190/194874593-f301ed8b-8c58-4507-aabb-54a826d54f56.png' alt="ugly_delicious_free_illustrations" /></a> | [Ugly Delicious](https://drawer.design/products/ugly-delicious/)  <br><br> ✔ Free <br><br>Format <br> • Sketch<br> • Adobe Illustrator<br> • Adobe Xd<br> • Adobe After Effects<br> • Figma<br> • Lottie<br> • SVG<br> • EPS<br> • PNG<br><br> ✌️Author <br> Drawer |
```

Comment:
- change the src link to `https://user-images.githubusercontent.com/74038190/195573490-a4751388-a786-4b57-b53d-df12c412064b.png` (the size is changed)

Reply:
This was not needed, since your previous one exactly matches my recommended one.